### ISSUE TITLE
Question on Readme instructions for installation.

### ISSUE THREAD

**Issue by mbribeiro** [Lelik13a/Zabbix-PyOra-ActiveCheck#4]
The second step in the installation says: "Create zabbix api user with read permissions on group, where databases hosts will be."

We got confused regarding this zabbix api user, if this user must be created in oracle database we want to monitor or in the server running the scripts. We also don't know what is this "group" that this user must have read permission. Is this api user a simple linux user we must create in the server that holds the agent or another user created in oracle just like in the step 1.

Thanks in advance.

Marcelo.

**Comment by dan-aksenov** [comment]
Agree, sounds confusing. Yet I always user same user as one running zabbix-agent.
Works fine.

**Comment by mbribeiro** [comment]
Great, so is this ok if I skip step 2 and simple use the "zabbix" user ?

**Comment by Lelik13a** [comment]
In second step we created zabbix user in web-interface. He will be read data from DB hosts and create items check list in /usr/lib/zabbix/cache/items-*.

**Comment by as007012** [comment]
Linux安装的zabbix.3.4.15版本，被监控端必须手动执行py脚本服务端才能展示数据。

**Comment by dmjaya** [comment]
12 Configure needed template and hosts items
is this step must or can ignore, If it is must could you please advice how to do it?

**Comment by Lelik13a** [comment]
This is active checks.
# Perform checks by items list and print additional verbose information for every check
pyora-active.py  --address 10.0.0.1 --database SID  -v
In this step data send on zabbix server to host.
Host and items must be exist.

**Comment by dmjaya** [comment]
Hello Lelik 
Thanks For replying , Im newer to Oracle monitoring,
Could you please send me the steps how to monitoring the Oracle DB using your scripts

Thanks

**Comment by Lelik13a** [comment]
Unfortunately, I worked for a long time with oracle monitoring. And now I have no opportunity to test the script.
Try to see the instructions of the original verification script https://github.com/bicofino/Pyora

**Comment by dmjaya** [comment]
![image](https://user-images.githubusercontent.com/13479490/56436783-a91b5e80-62fa-11e9-8665-9043107fc5bd.png)
do you have any idea above atachment

**Comment by Lelik13a** [comment]
Check step 7 and 8 - zabbix agent must have checks items:
```
UserParameter=pyora-discovery[*],/usr/lib/zabbix/externalscripts/pyora-discovery.py --address $1 --database $2 $3
UserParameter=pyora-items-list[*],/usr/lib/zabbix/externalscripts/pyora-items-list.py --zabbixurl "$1" --zabbixuser "$2" --zabbixpassword "$3" --hostname "$4" --address "$5" --database "$6"
```

**Comment by dmjaya** [comment]
I update the above file like this Please check is this ok?

UserParameter=pyora-discovery[*],/usr/lib/zabbix/externalscripts/pyora-discovery.py --address '192.168.10.10' $1 --database 'TEST1' $2 $3
UserParameter=pyora-items-list[*],/usr/lib/zabbix/externalscripts/pyora-items-list.py --zabbixurl'http://192.168.10.20/zabbix' "$1" --zabbixuser 'SYS' "$2" --zabbixpassword 'admin@123' "$3" --hostname "ol7" "$4" --address '192.168.10.10' "$5" --database 'TEST1' "$6"

**Comment by Lelik13a** [comment]
No.
$1, $2 and so - zabbix variables, you don't need edit them.
Just copy file oracle_pyora.conf to /etc/zabbix/zabbix_agentd.d/ directory and restart zabbix agent.
When a zabbix server requests an agent, instead of "$1" variables will be presented.
See step 10.

**Comment by dmjaya** [comment]
ist mena no need to change any $ values?

**Comment by Lelik13a** [comment]
Yes.
Try use zabbix_get.
https://www.zabbix.com/documentation/current/manual/concepts/get
Check agent settings. May be config not included.

**Comment by dmjaya** [comment]
Hi.
Thanks, @Lelik13a finally I manage to configure the script, Thanks Again. I need to reduce the interval time **800 to 10** please advice how to do it? **"<expression>{Template Pyora active send:check_active.nodata(800)}=1</expression>"**
Thanks.

**Comment by Lelik13a** [comment]
The period should not be less than 30 seconds because the timer process calculates this function only every 30 seconds.
https://www.zabbix.com/documentation/3.4/manual/appendix/triggers/functions
Verification period is determined by cron (cron.d/zabbix-pyora). You can set necessary time, but 1 minute is the minimal interval supported.
If you want to check the availability of the database, it is better to use other, separate checks.

**Comment by dmjaya** [comment]
Noted, but after that, I got below issue

![image](https://user-images.githubusercontent.com/13479490/57607641-24360480-7589-11e9-88cb-7b807df516ff.png)

**Comment by wang509020** [comment]
hi,dmjaya

![image](https://user-images.githubusercontent.com/50124188/116019295-7db07480-a676-11eb-8014-c2552ea751d3.png)
do you have any idea regarding above attachment
tell me ,thks

**Comment by dmjaya** [comment]
What is the zabbix version your using? 
if your using 4.6+ versions download the new shared template & change the trigger & item values according to original template.
